trillian/examples/deployment/docker/ctfe/Dockerfile

```diff
@@ -1,15 +1,20 @@
 FROM golang:buster as build
 
-ADD . /go/src/github.com/google/certificate-transparency-go
-WORKDIR /go/src/github.com/google/certificate-transparency-go
-
 ARG GOFLAGS=""
 ENV GOFLAGS=$GOFLAGS
 ENV GO111MODULE=on
-RUN go get ./trillian/ctfe/ct_server
+
+WORKDIR /build
+
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+COPY . .
```

Comment:
What does this do? Copies locally modified files to the image?

Reply:
It copies all of the tree into the image. Before this we've just set up the go dependencies, and then this operation copies in the tree. I'm not sure why the whole operation isn't just `COPY . .` to copy everything into the tree, and then `go build ./my/thing`. But this way a) works, b) it's clear (if a little verbose), and c) is consistent with the other Dockerfiles.